userguide/virtualmachines.rst

```diff
@@ -140,50 +142,45 @@ Virtual machine configuration options are described in
    | Screen # | Setting            | Value          | Description                                                                                   |
```

Comment:
Do we need to have "Screen #" here?  We are showing these in order, and the numbers are usually not shown in the UI, at least not when the user is on that screen.